docker.local/b0docker-compose-github.yml

```diff
@@ -26,26 +26,6 @@ services:
     links:
       - postgres:postgres
 
-  blobber:
-    image: blobber
-    environment:
-      - DOCKER= true
-    volumes:
-      - ../config:/blobber/config
-      - ./blobber${BLOBBER}/files:/blobber/files
-      - ./blobber${BLOBBER}/data:/blobber/data
-      - ./blobber${BLOBBER}/log:/blobber/log
-      - ./keys_config:/blobber/keysconfig
-      - ./blobber${BLOBBER}/data/tmp:/tmp
-    ports:
-      - 505${BLOBBER}:505${BLOBBER}
-      - 3150${BLOBBER}:3150${BLOBBER}
-    command: ./bin/blobber --port 505${BLOBBER} --grpc_port 3150${BLOBBER} --hostname 198.18.0.9${BLOBBER} --deployment_mode 0 --keys_file keysconfig/b0bnode${BLOBBER}_keys.txt --files_dir /blobber/files --log_dir /blobber/log --db_dir /blobber/data --minio_file keysconfig/minio_config.txt
-    networks:
-      default:
-      testnet0:
-        ipv4_address: 198.18.0.9${BLOBBER}
```

Comment:
Why did you remove this?

Reply:
it is implemented by `make local-run` instead .

Reply:
see https://github.com/0chain/blobber/blob/162c29b0f2b938160004354dfb770fdc7bcd235c/Makefile#L51

Reply:
only postgres and postgres-post are running inside docker. It is difficult to debug test code inside docker. so move them out from docker